features/step_definitions/networking.rb

```diff
@@ -1322,6 +1322,7 @@
 
 Given /^I save multus pod on master node to the#{OPT_SYM} clipboard$/ do | cb_name |
   ensure_admin_tagged
+  cb_name = "multus_pod" unless cb_name
```

Comment:
@asood-rh you can use this format instead `cb_name ||= "multus_pod"` or ` cb_name ||= :multus_pod`

Reply:
@asood-rh seeing that you are calling it like ```Given I save multus pod on master node to the :multuspod clipboard``` so to be consistent you can change that statement to `cb_name ||= :multspod`

Reply:
Done - https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3241/console